### PR TITLE
refactor: changed env var names

### PR DESCRIPTION
**Pull request by ical10**
## Describe your changes
Added missing env which will be removed once wagmi is fully integrated

## Issue ticket number / link
No issue ticket, minor fix

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -12,8 +12,11 @@ const nextConfig = {
   },
   publicRuntimeConfig: {
     appAuthURL: process.env.NEXTAUTH_URL ?? 'http://localhost:3000',
-    apiURL: process.env.DIGILAND_API_URL ?? 'http://localhost:3001',
+    apiURL: process.env.DIGILANDBALI_API_URL ?? 'http://localhost:3001',
+    providerKey: process.env.NEXT_PUBLIC_PROVIDER_KEY,
```
Comment (irmannmal) [inline next.config.js:16]: change to `web3ProviderKey: process.env.NEXT_PUBLIC_WEB3_PROVIDER_KEY`

File: next.config.js [hunk 2]
```diff
@@ -12,8 +12,11 @@ const nextConfig = {
...
+    providerURL: process.env.NEXT_PUBLIC_ALCHEMY_URL,
```
Comment (irmannmal) [inline next.config.js:17]: change to `web3ProviderURL: process.env.NEXT_PUBLIC_WEB3_PROVIDER_URL`

File: next.config.js [hunk 3]
```diff
@@ -12,8 +12,11 @@ const nextConfig = {
...
     contractAddress: process.env.NEXT_PUBLIC_LBSF_CONTRACT_ADDRESS,
```
Comment (irmannmal) [inline next.config.js:18]: change to `lbsfContractAddress: process.env.NEXT_PUBLIC_LBSF_CONTRACT_ADDRESS`

File: next.config.js [hunk 4]
```diff
@@ -12,8 +12,11 @@ const nextConfig = {
   },
   publicRuntimeConfig: {
     appAuthURL: process.env.NEXTAUTH_URL ?? 'http://localhost:3000',
-    apiURL: process.env.DIGILAND_API_URL ?? 'http://localhost:3001',
+    apiURL: process.env.DIGILANDBALI_API_URL ?? 'http://localhost:3001',
```
Comment (irmannmal) [inline next.config.js:15]: change to `apiURL: process.env.API_URL ?? 'http://localhost:3001',

**Review by irmannmal — CHANGES_REQUESTED** [review]

**Comment by irmannmal** [comment]
bor @ical10, don't forget to update [.env-template](https://github.com/digilandbali/digilandbali-web/blob/main/.env-template)

**Comment by ical10** [comment]
please recheck @irmannmal

**Review by irmannmal — APPROVED** [review]

**Pull request merged**